STEERING-COMMITTEE.md

```diff
@@ -1,32 +1,115 @@
+    1. The top three Companies contributing to Istio are eligible for
+    Contribution Seats, proportional to their contribution.
+        1. Each company is allocated one seat;
+        1. The remaining six seats are allocated based on project contribution,
+        with no Company exceeding 5 seats in total as outlined in this Charter.
+1. There shall be four **Community Seats** elected by the Istio contributors
+and community, beginning in August 2020.
+    1. Any [project member](ROLES.md#member) can self-nominate for the
+    election, or nominate another project member with their consent.
+    1. Elections shall use time-limited, Condorcet voting.
+    1. The following are eligible to vote for Community Seats:
+        1. [Project members](ROLES.md#member) who have had a merged PR in
+        the 12 months prior to the election; and
+        1. People who have submitted a voting exception form to the
+        Steering Committee, demonstrating contribution to the Istio project
+        that is of a non-code nature in the 12 months prior to the election,
+        and are granted a vote for the election by a simple majority vote of
+        the Steering Committee.
+    1. No Company can hold more than one Community Seat.
+    1. Community Seats are maintained for the term by the individual, even
+    if they change their Company affiliation.
+        1. If an individual changes company affiliation mid-term in a way
+        that is incompatible with the Company representation policies in
+        this Charter, the individual will be considered to have resigned
+        their seat and the first eligible runner-up from the previous
+        election will replace them for the duration of the term. If there
+        is no first runner-up eligible to serve in the as per these rules,
+        then the Steering Committee shall vote by simple majority to replace
+        the seat for the term.
+    1. Because the goal of Community Seats is to increase the perspectives on
+    the Steering Committee, employees of a Company that holds Contribution
+    Seats are ineligible to be elected to hold a Community Seat.
+1. A simple majority of Seats shall be sufficient to convene a meeting of the
+Steering Committee, one nominating, and the rest agreeing, over email. Meeting
+proposals shall be circulated to all Seats, and time, location, and medium shall
+be selected to be as convenient as possible. All Seats shall be given at least
+one week written notice that a meeting of the Steering Committee will be held.
+1. At all meetings of the Steering Committee convened under this charter, at
+least 60% of the Seats shall constitute a quorum for voting purposes. The vote
+of an affirmative vote of 60% of the Seats present at the time of the vote shall
+be the decision of the Steering Committee. However, any changes to the Steering
+Committee Charter shall require a vote of 80% of the Seats present at the time of
+the vote.
```

Comment:
Sorry it is not clear to me whether the vote has to be done at the SC meeting or can be offline.  Based on the reading, it seems to indicate members have to be present to vote.

Reply:
"time, location, and medium shall be selected to be as convenient as possible".

In our last meeting we chose to interpret this as allowing some or all voting to happen async, by e-mail, etc.

Reply:
Right, i just wanted to make sure that is reflected here as the charter has `Seats present at the time of the vote.`

Reply:
Ok, we are going to consider "participating in the vote" instead of "present at the time of the vote."